sdk/src/configs/markets.ts

```diff
@@ -0,0 +1,536 @@
+/*
```

Comment:
let's update our listing checklist so that we add new markets to sdk as well (even if it's in PR)

https://www.notion.so/gmxio/New-token-market-checklist-11203574745d8067b91de2cba9f52ad9#11203574745d80529ad0c8f5bea8f5cd

Reply:
Draft is here: https://www.notion.so/gmxio/New-token-market-checklist-pre-SDK-14b03574745d802c962ccb54c02a5964, move to main checklist once we merge